commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -268,3 +270,37 @@ def get_product_variant_by_course_run(self, cr_id: str) -> Optional[CTProductVar
             return None
 
         return matching_variant_list[0]
+
+    def create_return_for_order(self, order_id: str, order_version: str, order_line_id: str) -> CTOrder:
+        """
+        Creates refund/return for Commercetools order
+        Args:
+            order_id (str): Order ID (UUID)
+            order_version (str): Current version of order
+            order_line_id (str): ID of order line item
+        Returns (CTOrder): Updated order object or
+        Returns Exception: Error if update was unsuccesful.
+        """
+
+        try:
+            return_item_draft = ReturnItemDraft(
+                quantity=1,
```

Comment:
Instead of hardcoding the quantity here, Can we fetch the quantity from the order's lineItem's `quantity` property?

Reply:
@pshiu what's your take on this? I know we agreed that we could hard code the `quantity` value because it will always be 1 when creating a `ReturnItemDraft`.